src/Camelotia.Presentation.Avalonia/Views/ProviderView.xaml

```diff
@@ -3,10 +3,41 @@
              xmlns:views="clr-namespace:Camelotia.Presentation.Avalonia.Views"
              xmlns:interfaces="clr-namespace:Camelotia.Presentation.Interfaces;assembly=Camelotia.Presentation"
              xmlns:designTime="clr-namespace:Camelotia.Presentation.DesignTime;assembly=Camelotia.Presentation"
+             xmlns:reactiveui="http://reactiveui.net"
```

Comment:
This `xmlns` seems to be unused btw.

Reply:
Indeed, was leftover from some testing. Well spotted 👍